notebooks/Annotation and Continuous Learning Overview.ipynb

```diff
@@ -11,253 +11,91 @@
-      "      \"71\": \"left rib 1\",\n",
-      "      \"72\": \"left rib 2\",\n",
-      "      \"73\": \"left rib 3\",\n",
-      "      \"74\": \"left rib 4\",\n",
-      "      \"75\": \"left rib 5\",\n",
-      "      \"76\": \"left rib 6\",\n",
-      "      \"77\": \"left rib 7\",\n",
-      "      \"78\": \"left rib 8\",\n",
-      "      \"79\": \"left rib 9\",\n",
-      "      \"8\": \"right adrenal gland\",\n",
-      "      \"80\": \"left rib 10\",\n",
-      "      \"81\": \"left rib 11\",\n",
-      "      \"82\": \"left rib 12\",\n",
-      "      \"83\": \"right rib 1\",\n",
-      "      \"84\": \"right rib 2\",\n",
-      "      \"85\": \"right rib 3\",\n",
-      "      \"86\": \"right rib 4\",\n",
-      "      \"87\": \"right rib 5\",\n",
-      "      \"88\": \"right rib 6\",\n",
-      "      \"89\": \"right rib 7\",\n",
-      "      \"9\": \"left adrenal gland\",\n",
-      "      \"90\": \"right rib 8\",\n",
-      "      \"91\": \"right rib 9\",\n",
-      "      \"92\": \"right rib 10\",\n",
-      "      \"93\": \"right rib 11\",\n",
-      "      \"94\": \"right rib 12\",\n",
-      "      \"95\": \"left humerus\",\n",
-      "      \"96\": \"right humerus\",\n",
-      "      \"97\": \"left scapula\",\n",
-      "      \"98\": \"right scapula\",\n",
-      "      \"99\": \"left clavicula\"\n",
-      "    }\n",
-      "  },\n",
-      "  \"name\": \"my_vista\",\n",
-      "  \"network_arch\": \"monai_vista3d\",\n",
-      "  \"ngc_path\": \"\",\n",
-      "  \"ptm\": [\n",
-      "    \"543a531e-4533-4444-9664-f315a1e20645\"\n",
-      "  ],\n",
-      "  \"public\": false,\n",
-      "  \"read_only\": false,\n",
-      "  \"realtime_infer\": true,\n",
-      "  \"realtime_infer_request_timeout\": 60,\n",
-      "  \"train_datasets\": [\n",
-      "    \"62ed9367-56df-4813-b85a-b78b5e897096\"\n",
-      "  ],\n",
-      "  \"type\": \"monai\",\n",
-      "  \"version\": \"1.0.0\"\n",
-      "}\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "import json\n",
     "import requests\n",
-    "\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "af3efbe6",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
     "# API Endpoint and Credentials\n",
-    "monai_cloud_api = \"<MONAI Cloud API URL>\"\n",
-    "api_url = f\"{monai_cloud_api}/api/v1\"\n",
+    "host_url = \"<MONAI Cloud API URL>\"\n",
     "ngc_api_key = \"<NGC API Key>\"\n",
-    "\n",
+    "# Dicom Server\n",
+    "dicom_web_endpoint = \"<DICOMWeb address>\" # For example \"http://127.0.0.1:8042/dicom-web\".\n",
+    "dicom_client_id = \"<DICOMWeb user ID>\"    # If Authentication is enabled, then provide username\n",
+    "dicom_client_secret = \"<DICOMWeb secret>\" # If Authentication is enabled, then provide password\n",
+    "# inference image id\n",
+    "inference_image_id = \"<Image ID from OHIF>\""
```

Comment:
inference_image_id seems weird here because we're talking about credentials.  Maybe move it under the action: inference section in "Using Segment Everything"?

Reply:
same comments as below.

Reply:
Hi @zephyrie , I clicked resolve for other similar comments (such as epochs and timeout) and removed those parameters. But for the inference image id, as the limitation I said (all parameters to be changed should be in the same cell), we may still need to put it here. Hi @zephyrie @mingxin-zheng , do you have any suggestions?

Reply:
Hi @yiheng-wang-nv , it needs to be in this cell number because of the integration test on these notebook. Is my understanding correct?

Reply:
Yes. Papermill can override the notebook, but requires to create one (only one) cell with tag `parameters`. All variables should be in that cell.

Reply:
Can we generate the image id by doing the next image call?

Reply:
Feel free to reposition the nextimage cell for now.

Reply:
ok, let me modify it